### ISSUE TITLE
Beagle tutorial

### ISSUE THREAD

**Issue by victorrabelozup** [ZupIT/beagle-docs#652]
Today most of the users that looking for help come like:
"I'm trying to use Beagle and is not working",
"Beagle is just showing me an white screen with a retry but don't show the error".

The majority of this people have a decode error, I usually just go the the ScreenViewModel class and debug from there to catch the error, so I bring this discussion to the core team. We discuss an improvement in the framework and in the documentation.

The error log is not showed to the user because it need a logger to be configured, if is the user don't have this information he will not see the error and will be lost trying to debug.

So we suggest the docs to have more focus to the in the BeagleHelper library(in the initial section), this library was made to facilitate the first steps with Beagle but we think that it doesn't have enough mention and could be a easy way to at least show the error t the user, since the logger will be configured.

**Comment by carlossteinzup** [comment]
@victorrabelozup You are right in your sugestions. We are already updating the getting started section and a few other section on Docs and wii address the helpers options.
